esmvaltool/preprocessor/_regrid.py

```diff
@@ -141,6 +154,10 @@ def regrid(cube, target_grid, scheme):
         latitude (degrees) for a global, regular target grid.
     scheme : str
         The regridding scheme to perform, see `regrid.HORIZONTAL_SCHEMES`.
```

Comment:
This was already missing, but while we're changing stuff, it would be good to just mention the possible horizontal schemes, because HORIZONTAL_SCHEMES is not visible on on readthedocs:
https://esmvaltool.readthedocs.io/en/latest/codedoc2/esmvaltool.preprocessor.html#esmvaltool.preprocessor.regrid

Reply:
...aaand done as well! well spotted, man